.github/workflows/stalebot.yml

```
@@ -0,0 +1,26 @@
+name: Close stale Issues and PRs
+
+permissions:
+  issues: write
+  pull-requests: write
+
+on:
```

👇 Will squash the warning that YAMLLint is raising on this PR 🙂 

```suggestion
on: # yamllint disable-line rule:truthy
```

---

oops; it auto-merged. Thanks, didn't know that. Should send out another update soon.